openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.component.tsx

```diff
@@ -153,181 +102,180 @@ const AddGlossary = ({
       layout={PageLayoutType['2ColRTL']}
       pageTitle={t('label.add-entity', { entity: t('label.glossary') })}
       rightPanel={fetchRightPanel()}>
-      <div className="tw-form-container">
+      <div className="tw-form-container glossary-form">
         <Typography.Title data-testid="form-heading" level={5}>
           {header}
         </Typography.Title>
         <div className="tw-pb-3" data-testid="add-glossary">
-          <Field>
-            <label className="tw-block tw-form-label" htmlFor="name">
-              {requiredField(`${t('label.name')}:`)}
-            </label>
-
-            <input
-              className="tw-form-inputs tw-form-inputs-padding"
-              data-testid="name"
-              id="name"
+          <Form form={form} onFinish={handleSave}>
+            <Form.Item
+              label={t('label.name')}
+              labelCol={{ span: 24 }}
               name="name"
-              placeholder={t('label.name')}
-              type="text"
-              value={name}
-              onChange={handleValidation}
-            />
-
-            {showErrorMsg.name
-              ? ADD_GLOSSARY_ERROR[AddGlossaryError.NAME_REQUIRED]
-              : showErrorMsg.invalidName
-              ? ADD_GLOSSARY_ERROR[AddGlossaryError.NAME_INVALID]
-              : null}
-          </Field>
-          <Field>
-            <label className="tw-block tw-form-label" htmlFor="display-name">
-              {`${t('label.display-name')}:`}
-            </label>
-
-            <input
-              className="tw-form-inputs tw-form-inputs-padding"
+              rules={[
+                {
+                  required: true,
+                  message: `${t('message.field-text-is-required', {
+                    fieldText: t('label.name'),
+                  })}`,
+                },
+                {
+                  pattern: ENTITY_NAME_REGEX,
+                  message: `${t('label.field-invalid', {
```

Comment:
We should show relevant message like allowed character are this so that user gets clear picture of what is wrong with the name

Reply:
Updated the message